MmSupervisorPkg/MmSupervisorPkg.dsc

```diff
@@ -20,6 +20,8 @@
   BUILD_TARGETS                  = DEBUG|RELEASE
   SKUID_IDENTIFIER               = DEFAULT
 
+!include MdePkg/MdeLibs.dsc.inc

```

Comment:
This works, but I think we don't want to encourage use of MdeLibs.dsc.inc. It has been an edk2 compromise. We can directly add the stack cookie logic here.